### ISSUE TITLE
Uploaded code - only one leg is walking on the robot?

### ISSUE THREAD

**Issue by IVIACH1NE** [RadioShackCorp/2770168-MakeItRobotics-Starter-Kit#5]
Not too sure how this happened, but only the left leg is walking. 
Could there be an error in the code?

**Comment by bloodshot83** [comment]
I fought with that same problem for over a month.
What it ended up being for me was the IR sensors taking in to much light from the surrounding room. I also found it could be caused by the wires being crossed going to the sensors.

Thank you again
  Reed P R  2

On May 4, 2016, at 7:51 PM, IVIACH1NE <notifications@github.com<mailto:notifications@github.com>> wrote:

Not too sure how this happened, but only the left leg is walking.
Could there be an error in the code?

##

**Comment by IVIACH1NE** [comment]
I assumed it was the sensors. It was the only variable, everything was wired properly. I disconnected the sensors and hooked the primary power for the motors into the same port. Gets the job done. Not as efficiently....but it moves ;)

Thanks for the answer.

**Comment by bloodshot83** [comment]
Yeah... That's how I ended up having it "work" after spending the time with the data sheets. I will suggest building the line following one and use straws cut to a quarter inch pieces and slip them over the sensors. It will work well and it's a lot more satisfying seeing it actually do what it was intended too. lol
Thank you again
  Reed P R 2

On May 4, 2016, at 8:14 PM, IVIACH1NE <notifications@github.com<mailto:notifications@github.com>> wrote:

I assumed it was the sensors. It was the only variable, everything was wired properly. I disconnected the sensors and hooked the primary power for the motors into the same port. Gets the job done. Not as efficiently....but it moves ;)

Thanks for the answer.

##

**Comment by IVIACH1NE** [comment]
Do you know what I can edit to make the bot not sensor-based? Essentially, just make it a walker ?

**Comment by bloodshot83** [comment]
I would, not saying that it is the best or right way but a time delay. Kinda like for blinking a led. Just for left and for right motors instead.

Thank you again
  Reed P R 2

On May 4, 2016, at 8:44 PM, IVIACH1NE <notifications@github.com<mailto:notifications@github.com>> wrote:

Do you know what I can edit to make the bot not sensor-based? Essentially, just make it a walker ?

##
